site/docs/v1/tech/apis/_system-configuration-request-body.adoc

```diff
@@ -4,7 +4,7 @@
 [field]#systemConfiguration.auditLogConfiguration.delete.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.7.0#::
 Whether or not FusionAuth should delete the Audit Log based upon this configuration. When `true` the [field]#auditLogConfiguration.delete.numberOfDaysToRetain# will be used to identify audit logs that are eligible for deletion. When this value is set to `false` audit logs will be preserved forever.
 
-[field]#systemConfiguration.auditLogConfiguration.delete.numberOfDaysToRetain# [type]#[Integer]# [optional]#Optional# [since]#Available since 1.7.0#::
+[field]#systemConfiguration.auditLogConfiguration.delete.numberOfDaysToRetain# [type]#[Integer]# [optional]#Optional# [default]#defaults to `365`# [since]#Available since 1.7.0#::
```

Comment:
I think this default is new in a recent release. 

We should document when it was defaulted, my guess is prior to it being defaulted it was validated when `delete.enabled` was `true`, but listed as optional since it was not validated until enabled. 

For example:

> Prior to version X this value was required when [field]#delete.enabled# was equal to `true`. 

Same for any other new defaults.